src/components/Rockets.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import '../Styles/Rocket.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { cancelReservation, fetchRockets, reserveRocket } from '../redux/rocketSlice';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const status = useSelector((state) => state.rockets.status);
+  const error = useSelector((state) => state.rockets.error);
```

Comment:
- [ ] Great progress on your project! I noticed some code redundancy, and as a developer, it's essential to avoid repetition. Since all the items originate from the same Redux slice, it would be beneficial to retrieve them all at once, rather than fetching them individually. Please consider making this adjustment:

`const { rockets, status, error } = useSelector((state) => state.rockets)`